docs/index.asciidoc

```diff
@@ -222,5 +223,15 @@ of your ruby script file defined in `path`.
 Tag to add to events in case the ruby code (either inline or file based)
 causes an exception.
 
+[id="plugins-{type}s-{plugin}-tag_with_exception_message"]
+===== `tag_with_exception_message`
+
+* Value type is <<boolean,boolean>>
+* Default value is `false`
+
+If `true` adds a tag to the event that's the concatenation of <<plugins-{type}s-{plugin}-common-tag_with_exception_message>>
```

Comment:
Here's the problem... There is no anchor for <<plugins-{type}s-{plugin}-common-tag_with_exception_message>> to resolve to.

Reply:
By bad, an error in cut and paste